src/main/java/com/checkmarx/flow/config/GitLabProperties.java

```diff
@@ -9,10 +9,15 @@
 @Validated
 public class GitLabProperties extends RepoProperties {
 
+    public static final String MERGE_NOTE = "%s/projects/%s/merge_requests/%s/notes";
```

Comment:
Can it be private?

Reply:
This change will be included in next PR.